packages/app/src/ui/organisms/ProgrammeProposalForm.spec.tsx

```diff
@@ -1,9 +1,185 @@
-import { render } from "@testing-library/react";
+/**
+ * @jest-environment jsdom
+ */
 
+import { getByText, waitFor } from "@testing-library/dom";
+import { act, fireEvent, render, screen } from "@testing-library/react";
+import { I18nextProvider } from "react-i18next";
+import { dedent } from "ts-dedent";
+
+import { i18n } from "../../i18n";
+import { settings } from "../../types";
 import { ProgrammeProposalForm } from "./ProgrammeProposalForm";
 
 describe(ProgrammeProposalForm.name, () => {
-  it("does something", () => {
-    expect(true);
+  it("can be filled and submitted", async () => {
+    const form: settings.Form = {
```

Comment:
This place and `docsComponentProps` in packages/app/src/ui/organisms/ProgrammeProposalForm.docs.tsx look have the same content and they are both used to render _example_ of component. Why don't move them in separate file? When the component will change and you will want to add something to those settings it will be easier to change.
Isn't it DRY?